extensions/jdbc/jdbc-derby/jdbc-derby-deployment/src/main/java/io/quarkus/jdbc/derby/deployment/JDBCDerbyProcessor.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.jdbc.derby.deployment;
+
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+
+public class JDBCDerbyProcessor {
```

Comment:
While not required, we tend to have the reflections declaration in the same class. But, I like your structure, it's clean.
Maybe, you should mention in the javadoc the other classes containing BuildSteps.

Reply:
For the JDBC modules this seemed to be the pattern used, I can add some javadoc elaborating a bit on this though

Reply:
Considering how small the two are, I would merge everything in the Processor class.